tests/e2e/functional_test.go

```diff
@@ -151,7 +151,7 @@ var (
 	ISBServiceSpecExcludedField = numaflowv1.InterStepBufferServiceSpec{
 		Redis: nil,
 		JetStream: &numaflowv1.JetStreamBufferService{
-			Version: "2.9.8",
+			Version: updatedJetstreamVersion,
 			Persistence: &numaflowv1.PersistenceStrategy{
 				VolumeSize: &volSize,
 			},
```

Comment:
nit: do you mind updating this file to change the use of the word "excluded" to "direct apply" or something like that?